cmd/cli/job/describe.go

```diff
@@ -75,6 +75,10 @@ func NewDescribeCmd() *cobra.Command {
 			return o.run(cmd, args, api)
 		},
 	}
+
+	jobCmd.SilenceUsage = true
+	jobCmd.SilenceErrors = true
```

Comment:
instead of doing this in each and every command, we can just silence the root command
https://github.com/bacalhau-project/bacalhau/blob/1ca678c752371f790e84220e6061e63d3aed5e32/cmd/cli/root.go#L110-L118

Tested this works:
```

func Execute(ctx context.Context) {
	rootCmd := NewRootCmd()
	rootCmd.SetContext(ctx)
	rootCmd.SilenceErrors = true
	rootCmd.SilenceUsage = true
	if err := rootCmd.Execute(); err != nil {
		util.Fatal(rootCmd, err, 1)
	}
}
```

Reply:
Acknowledge. Fixing it !

Reply:
Actually i did try that and it does not work a root command layer.  You unfortunately have to do it at all layers.